### PR TITLE
Arm Virtual Hardware Guide to commission a Matter device over Bluetooth and control over Wi-Fi.

### PR DESCRIPTION
**Pull request by pareenaverma**
Adding a detailed guide to demonstrate the usage of Arm Virtual Hardware for commissioning a Matter device over Bluetooth and controlling it over Wi-Fi.

### CODE REVIEW

File: docs/guides/arm_virtual_hardware_ble-wifi-commissioning.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+7. Disable the Ethernet device
+
+```console
+sudo ip link set dev eth0 down
+```
+
+## Commissioning the lighting device to the AVH virtual Wi-Fi network and turning the light on and off
+
+Run the following commands on the `ubuntu-chip-tool` virtual instance
```
[the remark this answers is not in the document]
Comment (pareenaverma) [reply docs/guides/arm_virtual_hardware_ble-wifi-commissioning.md:155]: Hi, we expect to add additional guides/append to this guide in the coming months for using AVH for development of Matter stack which will include the Python controller as well.

**Comment by andy31415** [comment]
@pareenaverma - needs restyle and spellcheck fixes.

File: docs/guides/arm_virtual_hardware_ble-wifi-commissioning.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+```console
+sudo ip link set dev eth0 down
+```
+
+19. Run lighting-app example.
+
+```console
+./out/debug/chip-lighting-app --wifi
+```
```
Comment (jmeg-sfy) [inline docs/guides/arm_virtual_hardware_ble-wifi-commissioning.md:116]: Having the wifi/ble is cool but do we have the equivalent with just Ethernet IP ?

**Comment by andy31415** [comment]
@pareenaverma - this requires restyle.
You can either fetch the commit from restyle or do `./scripts/helpers/restyle-diff.sh`

**Comment by andy31415** [comment]
@pareenaverma also needs misspell fixes

**Pull request closed without merging**